src/state_migration/nv17/tests/miner_tests.rs

```diff
@@ -0,0 +1,571 @@
+// Copyright 2019-2023 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+#[cfg(test)]
+mod tests {
```

Comment:
Why have you moved the tests here?

Reply:
To create an exception for `'src/state_migration/nv17/tests/'` when checking banned dependencies under PR #3110.

Reply:
Let's change the script rather than change the code.